# Message for defining a sequence
# actual position in the sequence
int32 sequence_number
# overall lenght of the sequence
int32 length